tests/newfeatures/ssh-key.feature

```diff
@@ -13,12 +13,12 @@ Feature: SSH Keys
 
   @vcr ssh-keys_add
   Scenario: Add an SSH key
-    When I run "terminus ssh-key:add --file=tests/config/dummy_key.pub"
+    When I run "terminus ssh-key:add ./tests/config/dummy_key.pub"
```

Comment:
I'm not sure if we ought to or not, I was just wondering if it was something that would render us some benefit. Please remove if you think not.